packages/postgraphile-core/__tests__/integration/__snapshots__/mutations.test.js.snap

```diff
@@ -98,7 +98,10 @@ Object {
             "inclusive": false,
             "value": 500,
           },
-          "start": null,
+          "start": Object {
+            "inclusive": false,
+            "value": null,
+          },
```

Comment:
It seems to me like the meanings of these are equivalent (i.e. "there is no lower bound"); as such I think we should stick with the existing structure or risk this being a breaking change for some users.

Reply:
ahh much better, I didn't notice the whole Bound the was already nullable